services/clojars/clojars-base.js

```diff
@@ -0,0 +1,65 @@
+'use strict'
+
+const Joi = require('joi')
+const { version: versionColor } = require('../color-formatters')
+const { nonNegativeInteger } = require('../validators')
+const { BaseJsonService, InvalidResponse } = require('..')
+
+const clojarsSchema = Joi.object({
+  downloads: nonNegativeInteger,
+  latest_version: Joi.string().required(),
+  latest_release: Joi.string().required(),
+}).required()
+
+class BaseClojarsService extends BaseJsonService {
+  async fetch({ clojar }) {
+    const url = `https://clojars.org/api/artifacts/${clojar}`
```

Comment:
If there's API documentation, can you add a comment with a link? For example:

`// Clojars Doc: https://....`

Reply:
added comment to Clojars API reference on wiki